packages/big-design-icons/scripts/svgr-flags.config.js

```diff
@@ -31,7 +35,7 @@ module.exports = {
     };
 
     BREAK
-    const FlagIconWithForwardedRef = forwardRef<SVGSVGElement, FlagIconProps>((iconProps, ref) => <FlagIcon aria-hidden={iconProps.title ? undefined : true} {...iconProps} svgRef={ref} />);
```

Comment:
At this point the "default" flag title wasn't available, hence `aria-hidden` was set to true even though flags have always a title.